src/olympia/landfill/serializers.py

```diff
@@ -0,0 +1,185 @@
+import random
+
+from rest_framework import serializers
+
+from olympia.amo.tests import user_factory, addon_factory
+from olympia import amo
+from olympia.addons.forms import icons
+from olympia.addons.models import AddonUser, Preview
+from olympia.addons.utils import generate_addon_guid
+from olympia.amo.tests import version_factory
+from olympia.constants.applications import APPS, FIREFOX
+from olympia.constants.base import (
+    ADDON_EXTENSION,
+    ADDON_PERSONA,
+    STATUS_PUBLIC
+)
+from olympia.landfill.collection import generate_collection
+from olympia.landfill.generators import generate_themes
+from olympia.reviews.models import Review
+from olympia.users.models import UserProfile
+
+
+class GenerateAddonsSerializer(serializers.Serializer):
+    count = serializers.IntegerField(default=10)
+
+    def create_addons(self):
+        """Creates 10 addons all named Ui-Addon.
+
+        These addons will not be featured and will all belong to the user
+        uitest.
+
+        They will all have 1 preview and 2 reviews.
+
+        """
+        for _ in range(10):
+            default_icons = [x[0] for x in icons() if x[0].startswith('icon/')]
+            addon = addon_factory(
+                status=STATUS_PUBLIC,
+                type=ADDON_EXTENSION,
+                average_daily_users=7000,
+                users=[UserProfile.objects.get(username='uitest')],
+                average_rating=3,
+                description=u'My Addon description',
+                file_kw={
+                    'hash': 'fakehash',
+                    'platform': amo.PLATFORM_ALL.id,
+                    'size': 42,
+                },
+                guid=generate_addon_guid(),
+                icon_type=random.choice(default_icons),
+                name=u'Ui-Addon',
+                public_stats=True,
+                slug='ui-test-2',
+                summary=u'My Addon summary',
+                tags=['some_tag', 'another_tag', 'ui-testing',
+                      'selenium', 'python'],
+                total_reviews=777,
+                weekly_downloads=22233879,
+                developer_comments='This is a testing addon.',
+            )
+            Preview.objects.create(addon=addon, position=1)
+            Review.objects.create(addon=addon, rating=5, user=user_factory())
+            Review.objects.create(addon=addon, rating=3, user=user_factory())
+            addon.reload()
```

Comment:
I don't think `addon.reload` should be called, the object is not being used afterwards where we'd need that.